src/parse.coffee

```diff
@@ -33,8 +33,7 @@ parse = (source, callback) ->
       err = new Error(err.message)
 
     if apiElements
-      apiElements = apiElements.toRefract()
-      apiElements.content = apiElements.content.concat(annotations)
+      apiElements.content.unshift(warning) if warning
```

Comment:
If it helps explaining the problem, here is a small code-example using uptown and a failing test:

#### `test/element.js`

```js
var createClass = require('../lib/uptown').createClass;


module.exports = function(registry) {
  return createClass({
    constructor: function() {}
  });
};
```

```diff
diff --git a/test/functional/create-class-test.js b/test/functional/create-class-test.js
index bd0c53d..38d5475 100644
--- a/test/functional/create-class-test.js
+++ b/test/functional/create-class-test.js
@@ -1,5 +1,6 @@
 var _ = require('lodash');
 var expect = require('../test-helper').expect;
+var Element = require('../element')();
 
 var createClass = require('../../lib/uptown').createClass;
 
@@ -38,4 +39,16 @@ describe('Creating Classes', function() {
     foo.x = 2;
     expect(foo.x).to.equal(2);
   })
+
+  it('can check if instances are instances', function() {
+    var LocalElement = require('../element')();
+    var instance1 = new Element();
+    var instance2 = new LocalElement();
+
+    expect(instance1).to.be.instanceof(Element);
+    expect(instance2).to.be.instanceof(Element);
+  });
 });
```